CHANGELOG.rst

```diff
@@ -1,5 +1,8 @@
 Changelog
 =========
+* 0.8.2 (March 7th, 2022)
```

Comment:
You may come up against the original maintainers when you try get this merged back into trunk but it's fine for our branch

Reply:
Yeah I wasn't planning on opening a PR on the original package. Figured keeping to our forked version would be ok.

Thanks :)